picard/formats/id3.py

```diff
@@ -494,6 +494,19 @@ def _get_tags(self, filename):
         except mutagen.id3.ID3NoHeaderError:
             return compatid3.CompatID3()
 
+    def _load_preserved_config(self, metadata):
+        # Adding additional config to metadata to prevent unecessary saving
+        metadata['~config:rating_steps'] = config.setting['rating_steps']
+        metadata['~config:rating_user_email'] = config.setting['rating_user_email']
```

Comment:
Currently we are always saving it for the first time a save is being called on it and then skipping on saving it afterwards if no settings have been changed.

Me and @mineo discussed it over irc that we should avoid saving any tags to the file and that a first save regardless is the way to go.